src/app/clusters/ota-requestor/OTARequestorDriver.h

```diff
@@ -69,11 +69,21 @@ enum class IdleStateReason
     kInvalidSession,
 };
 
+// The current selected OTA Requestor timer to be running
+enum class SelectedTimer
+{
+    kPeriodicQueryTimer,
+    kWatchdogTimer,
+};
+
 // Interface class to abstract the OTA-related business logic. Each application
 // must implement this interface. All calls must be non-blocking unless stated otherwise
 class OTARequestorDriver
 {
 public:
+    using OTAUpdateStateEnum  = app::Clusters::OtaSoftwareUpdateRequestor::OTAUpdateStateEnum;
```

Comment:
I don't think these declarations are needed anymore, right?

Reply:
Not sure why sometimes my code deletions don't show up on my branch.